code/datums/mind.dm

```diff
@@ -110,7 +110,8 @@
 	var/mob/living/old_current = current
 	current = new_character								//associate ourself with our new body
 	new_character.mind = src							//and associate our new body with ourself
-	old_current.transfer_observers_to(current)			//transfer anyone observing the old character to the new one
+	if(old_current)
+		old_current.transfer_observers_to(current)			//transfer anyone observing the old character to the new one

```

Comment:
Actually, might be better to move this up before 111 and change it to current.transfer_observers_to(new_character)